tools/execute-selenium-tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_HOME="$CURRENT_PATH/.."
+FOXBOX_BINARY="$PROJECT_HOME/target/debug/foxbox"
+
+DEBUG_HOST=${BOX_LOCAL_NAME:-'localhost'}
+DEBUG_PORT=${BOX_PORT:-3331}
+DEBUG_PROFILE="$HOME/.local/share/foxbox-debug"
+
+# clean up residual profile
+rm -rf ${DEBUG_PROFILE}
```

Comment:
@JohanLorenzo doesn't the double quotes on the definition of DEBUG_PROFILE above cover this?